feature-libs/checkout/base/components/checkout-review-submit/checkout-review-submit.component.ts

```diff
@@ -102,6 +102,19 @@ export class CheckoutReviewSubmitComponent {
         ) {
           region = deliveryAddress.region.isocode + ', ';
         }
+        let numbers = '';
+        if (deliveryAddress.cellphone && deliveryAddress.phone) {
+          numbers =
+          `P: ${deliveryAddress.phone}
+          M: ${deliveryAddress.cellphone}`;
+          if(deliveryAddress.cellphone === deliveryAddress.phone) {
+            numbers = 'M: ' + deliveryAddress.cellphone;
+          }
+        } else if (deliveryAddress.cellphone && !deliveryAddress.phone) {
+          numbers = 'M: ' + deliveryAddress.cellphone;
+        } else if (!deliveryAddress.cellphone && deliveryAddress.phone) {
+          numbers =  'P: ' + deliveryAddress.phone;
+        }
```

Comment:
Huh, seeing this in multiple places makes me think this could be in a `*.utils.ts` file.

Reply:
Should the Phone/Mobile prefix be part of the translations file?

Reply:
Ah, sorry I didn't see this comment earlier.
I want to say... yes. As insignificant as it may be, we can't anticipate how other countries would want this.